services/certificate-api/src/utilities/cord.js

```diff
@@ -0,0 +1,105 @@
+// src/utilities/cord.js
+// CORD-related functions.
+
+import * as cord from '@cord.network/sdk'
+
+import { CORD_CHAIN_ADDRESS } from '../../config/index.js'
+
+// First, connect to the chain.
+await cord.init({ address: CORD_CHAIN_ADDRESS })
+
+// Retrieve the identities used to create schemas and credentials.
+// TODO: `Alice` works, but a newly created account doesn't. Why?
+const signer = cord.Identity.buildFromURI('//Alice', {
+	signingKeyPairType: 'sr25519',
+})
+// Create the schema and register it once `createClient` is called.
+const schemaContent = {
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "Basic Demo",
+	"description": "Test Demo Schema",
+	"$metadata": {
+		"version": "1.0.0",
+		"discoverable": true
+	},
+	"properties": {
+		"name": {
+			"type": "string"
+		},
+		"age": {
+			"type": "integer"
+		},
+		"gender": {
+			"type": "string"
+		},
+		"credit": {
+			"type": "integer"
+		}
+	},
+	"type": "object"
+}
+const spaceContent = {
```

Comment:
this can be technically dynamic, but for SunbirdRC usecase, can be static per deployment. If we don't put any dynamic part here, (something like instance id, or timestamp of creation), there may be issue of space identifier exists error from the chain. Hence if you look at the demo scripts, we added a 'uuid()' to the title to make it dynamic in every run.

Reply:
Actually, same applies to Schema. There should be an element of difference between two schemas.